GitUI/CommandsDialogs/RepoHosting/ForkAndCloneForm.Designer.cs

```diff
@@ -572,5 +613,8 @@ private void InitializeComponent()
         private System.Windows.Forms.FlowLayoutPanel flowLayoutPanel1;
         private System.Windows.Forms.Label _cloneInfoText;
         private System.Windows.Forms.TableLayoutPanel tableLayoutPanel5;
+        private System.Windows.Forms.Panel _cloneProtocolGroupBox;
+        private System.Windows.Forms.RadioButton _useHttpsRadioButton;
+        private System.Windows.Forms.RadioButton _useSshRadioButton;
```

Comment:
Please remove leading `_` for the new controls, it is quite atypical.
If you wish, can fix other controls' names in a separate commit.

Reply:
I thought it was odd, but when I added them initially I saw a few others had it so I added it to mine as well. I'll fix it and get the others in line as well.

Reply:
Added a commit to remove the leading `_` from each control.

Reply:
thank you